src/bin/hexyl.rs

```diff
@@ -209,6 +209,17 @@ fn run() -> Result<()> {
                 .hide(true)
                 .help("An alias for '--endianness=little'."),
         )
+        .arg(
+            Arg::new("character-table")
+                .long("character-table")
+                .value_name("FORMAT")
+                .value_parser(["codepage-437", "ascii-only"])
```

Comment:
I think we can define a default here? This way, we don't have to use `.unwrap_or(&String::from("ascii-only"))` below